jest.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const fs = require('fs');
+
+// Check for foremanReact files provided by Foreman and make available as module in Jest.
+const checkForForemanReact = (foremanLocations, foremanReact) => {
+  const currentDir = process.cwd();
+
+  let foremanReactFullPath;
+  foremanLocations.forEach((relativeForemanPath) => {
+    if (fs.existsSync(path.join(currentDir, relativeForemanPath))) {
+      const fullPath = path.join(currentDir, relativeForemanPath, foremanReact);
+      if (fs.existsSync(fullPath)) foremanReactFullPath = fullPath;
+    }
+  });
+  return foremanReactFullPath;
+};
+
+const foremanReactRelative = 'webpack/assets/javascripts/react_app';
+const possibleForemanLocations = ['./foreman', '../foreman', '../../foreman'];
+const notFound = 'Foreman directory can not be found! These tests require Foreman to be present ' +
+'in either a parent, sibling, or child directory relative to Katello and contain the expected ' +
```

Comment:
Was about to ask who in their right mind would put the foreman directory inside of Katello, but then saw the other PR comments :laughing: